src/reporting/private/iam.ts

```diff
@@ -105,6 +105,23 @@ export function createRoleForQuicksightCustomResourceLambda(
       ],
     }),
 
+    new PolicyStatement({
+      effect: Effect.ALLOW,
+      resources: [
+        `${arnPrefix}:folder/clickstream_*`,
```

Comment:
Let's extract `clickstream_` as a constant named QS_FOLDER_PREFIX. And use it to generate the folder id.

Reply:
Using existing shared variables: `QUICKSIGHT_RESOURCE_NAME_PREFIX`